script.js

```diff
@@ -0,0 +1,12 @@
+var links = document.getElementsByTagName("a");
```

Comment:
nice scripting here! try to avoid using the 'var' keyword as well for JS variables and stick with either 'let' or 'const'. Variable scope gets a bit more wonky when using var - [here is an article that explains more](https://codeburst.io/javascript-var-let-or-const-which-one-should-you-use-2fd521b050fa)!